week2/homework/src/index.js

```diff
@@ -1,3 +1,78 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+let fs = require ('fs');
+console.log(process.argv);
+
+let args = process.argv.slice(2);
+//console.log(args);
+
+let command = args[0];
+let todoItem = args[1]
+
+if(command === 'help') {
+   help(todoItem)
+} else if (command === 'add') {
+    add(todoItem)
+} else if (command === 'list') {
+    list(todoItem)
+} else if (command === 'remove') {
+    remove(todoItem)
+} else if (command === 'reset') {
+    reset(todoItem)
+}
```

Comment:
The case where user provides invalid command or no command should be considered.

Reply:
i will correct it, thanks.